docs/reference/releases.rst

```diff
@@ -29,6 +29,10 @@ Ubuntu Server / Desktop
      - 6.14
      - generic
 
+.. note::
+
+  The LTS versions of Ubuntu support newer kernel versions. Those are available as Hardware Enablement (HWE) kernels; see `Ubuntu kernel lifecycle`_.
```

Comment:
Maybe say "The LTS releases of Ubuntu support newer kernel versions." to avoid repeating 'versions' twice